### PR TITLE
base.yaml: Remove knitting_tools, default outfitting_tools to use kni…

### LINKED ISSUE #3186: double repair for knitting needles causing workorders to fail
This has just started.  Here is my tool setup and log below with double repair:

outfitting_belt:
  name: outfitter's belt
  items:
    - yardstick
    - scissors
    - pointed awl
    - sewing needles
    - knitting needles
    
outfitting_tools:
- knitting needles
- yardstick
- scissors
- sewing needles
#- hide scraper
- slickstone
- pointed awl
- stamp

Here is log:

[workorders]>untie my knitting needles from my outfitter's belt

You remove some polished steel knitting needles from your outfitter's belt.

[workorders]>give clerk

A clerk looks over the needles and says, "That will cost 6 Kronars to repair.  Just give it to me again if you want, and I'll have it ready in 1 roisaen."

[workorders]>give clerk

You hand the clerk 6 Kronars and he gives you back a repair ticket.  clerk says, "I should be having that done for you in about 1 roisaen.  Don't lose this ticket!  You must have it to reclaim your needles."

[workorders]>stow ticket

You put your ticket in your leather pack.

[workorders]>untie my yardstick from my outfitter's belt

You remove a yardstick inlaid with driftwood and bits of shell from your outfitter's belt.

[workorders]>give clerk

A clerk says, "There isn't a scratch on that, so there's nothing to repair."

[workorders]>tie my yardstick to my outfitter's belt

You attach a yardstick inlaid with driftwood and bits of shell to your outfitter's belt.

[workorders]>untie my scissors from my outfitter's belt

You remove some scissors with glaes cutting edges and decorative finger rings from your outfitter's belt.

[workorders]>give clerk

A clerk says, "There isn't a scratch on that, so there's nothing to repair."

[workorders]>tie my scissors to my outfitter's belt

You attach some scissors with glaes cutting edges and decorative finger rings to your outfitter's belt.

[workorders]>untie my sewing needles from my outfitter's belt

You remove some ribbed steel sewing needles from your outfitter's belt.
Binu attaches some polished steel knitting needles to his seamstress' belt.

[workorders]>give clerk

A clerk says, "There isn't a scratch on that, so there's nothing to repair."

[workorders]>tie my sewing needles to my outfitter's belt

You attach some ribbed steel sewing needles to your outfitter's belt.

[workorders]>get my slickstone

You get a polished alabaster slickstone from inside your haversack.

[workorders]>give clerk

A clerk says, "There isn't a scratch on that, so there's nothing to repair."

[workorders]>put my slickstone in my haversack

You put your slickstone in your haversack.

[workorders]>untie my pointed awl from my outfitter's belt

You remove a steel pointed awl from your outfitter's belt.
Binu goes west.

[workorders]>give clerk

A clerk says, "There isn't a scratch on that, so there's nothing to repair."

[workorders]>tie my pointed awl to my outfitter's belt

You attach a steel pointed awl to your outfitter's belt.

[workorders]>get my stamp

You get a basic steel stamp from inside your haversack.

[workorders]>give clerk

A clerk looks over the stamp and says, "That will cost 16 Kronars to repair.  Just give it to me again if you want, and I'll have it ready in 2 roisaen."

[workorders]>give clerk

You hand the clerk 16 Kronars and he gives you back a repair ticket.  clerk says, "I should be having that done for you in about 2 roisaen.  Don't lose this ticket!  You must have it to reclaim your stamp."

[workorders]>stow ticket

You put your ticket in your leather pack.

[workorders]>untie my knitting needle from my outfitter's belt

Untie what?

[workorders]>get my knitting needle

What were you referring to?

[workorders: You seem to be missing: knitting needle]

[hunting-buddy: ***STATUS*** EXECUTE go2 2391]

This ends up with two clerk repair tickets in pack.  So next time when workorders is called:

[sew]>untie my knitting needle from my outfitter's belt

Untie what?

[sew]>get my knitting needle

What were you referring to?

[sew: You seem to be missing: knitting needle]

--- Lich: sew has exited.

### PR DESCRIPTION
**Pull request by Sarvatt**
…tting needles.

Closes #3186

### CODE REVIEW

**Comment by Sarvatt** [comment]
workorders.lic will still recognize knitting_tools and honor it, eventually we can just do this after a validate check is added
```--- a/workorders.lic
+++ b/workorders.lic
@@ -92,7 +92,7 @@ class WorkOrders
         exit
       end
       @outfitting_room = @settings.outfitting_room
-      tools = (@settings.outfitting_tools + @settings.knitting_tools).uniq
+      tools = @settings.outfitting_tools
       @belt = @settings.outfitting_belt
       case item['chapter']
       when 4, 2, 3

File: profiles/base.yaml [hunk 1]
```diff
@@ -437,9 +437,8 @@ forging_tools:
 - tong
 - bellow
 - shovel
-knitting_tools:
```
Comment (rpherbig) [inline profiles/base.yaml]: ❌ If you're going to remove it here, please also remove it from base-empty: https://github.com/rpherbig/dr-scripts/blob/fc402777aac0c7bf8bb0ecf431293e5b54fcc227/profiles/base-empty.yaml#L28

❓ Speaking of which, I noticed `outfitting_tools` isn't defined in base-empty, could you add it?

**Comment by Sarvatt** [comment]
sorry for the delay, fixed now.

**Comment by Sarvatt** [comment]
I am going to do a few commits to test fixing the tests, sorry in advance

File: test/test_valid_yaml.rb [hunk 2]
```diff
@@ -17,6 +17,7 @@ def test_all_settings_in_base_yaml
 
     # Profiles using non-repo settings
     custom_profiles = %w[Chuno Erendel Shagium Crannach Dankmar Melborne Paeriluno Valkiss Ssarek Qetu Ugsy Valkiss Dartellum Leustyin Dijkstra Quillith]
+    shitty_settings = "knitting_tools"
```
Comment (rpherbig) [inline test/test_valid_yaml.rb]: 💡 Would it be easier just to fix each of the YAMLs currently in the repo?

Comment (Sarvatt) [reply test/test_valid_yaml.rb]: done now.

**Comment by Sarvatt** [comment]
Happy with this PR now.

**Pull request merged**